### ISSUE TITLE
Pick a license to use for this repo

### ISSUE THREAD

**Issue by msdisme** [open-infrastructure-labs/telemetrywg#3]
I did not pick on when I created the repo  - may be done via settings by Raja or Marcel or by me if we get consensus on which to use.

**Comment by RiRa12621** [comment]
![Screenshot 2020-12-16 at 15 38 51](https://user-images.githubusercontent.com/14996407/102362539-cebba800-3fb4-11eb-8ceb-aa42d1a6a956.png)
Apache 2.0 is probably good enough for most use cases that we face.

**Comment by fungi** [comment]
I agree, a permissive license like Apache, MIT/Expat, LGPL, or ISC makes a good default. Most Open Infrastructure Foundation projects use the Apache License Version 2.0, since some lawyers seemed convinced the additional patent protections it provides are useful when organizations with large patent profiles want to collaborate. Concern has been expressed in the past that copyleft licenses (e.g. GPL) could deter corporate contribution in particular.

Note that just because you assert a default license under which you expect contents to be contributed, that doesn't mean all contents in the repository must use it. It does, however, mean that you want to be very aware of and closely track any nonconforming content, documenting clearly that it's under a different license and checking carefully that you're not combining incompatible licenses in the way these contents get used (aggregation usually poses no concern, while library linking or importing can).

When picking a license, you also need to be sure to pay attention to and follow its instructions for how it should be applied. For Apache License Version 2.0, this guidance is documented at https://www.apache.org/licenses/LICENSE-2.0.html#apply .
